Dockerfile

```diff
@@ -38,3 +38,4 @@ WORKDIR /app
 
 RUN yarn install && yarn upgrade
 RUN passenger-config compile-nginx-engine --connect-timeout 60 --idle-timeout 60
+RUN rake assets:precompile && rake tmp:cache:clear
```

Comment:
This is the main insight to put assets inside the Docker container. This way we can push to production without running bundler / yarn in production.